extract_endpoint/tests/test_endpoint.py

```diff
@@ -93,47 +93,6 @@ def test_timestamp_no_file(test_client: testing.FlaskClient) -> None:
     assert get_return.status_code == HTTPStatus.BAD_GATEWAY
 
 
-def check_file_in_azure(azure_service: blob.BlockBlobService,
-                        azure_emulator_coords: azure_utils.StorageCoordinates,
-                        filename: str,
-                        contents: str) -> None:
-
-    assert filename in [blob.name for blob in azure_service.list_blobs(azure_emulator_coords.container)]
-    actual_blob = azure_service.get_blob_to_text(azure_emulator_coords.container, filename)
-    assert actual_blob.content == contents
-
-
-def test_upload_with_timestamp(azure_emulator_coords: azure_utils.StorageCoordinates,
-                               test_client: testing.FlaskClient,
-                               azure_service: blob.BlockBlobService,
-                               sample_timestamp: str,
-                               sample_salt: str,
-                               sample_zipfile_signature: str,
-                               sample_file_contents: str,
-                               sample_filenames: str,
-                               sample_zipfile: io.BytesIO) -> None:
-
-    post_return = test_client.post('/upload_file', data=dict(salt=sample_salt, signature=sample_zipfile_signature,
-                                                             timestamp=sample_timestamp,
-                                                             file=(sample_zipfile, 'zipfile')))
-    assert post_return.status_code == HTTPStatus.CREATED
-    check_file_in_azure(azure_service, azure_emulator_coords, endpoint.TIMESTAMP_FILENAME, sample_timestamp)
-    for name, contents in zip(sample_filenames, sample_file_contents):
-        check_file_in_azure(azure_service, azure_emulator_coords, name, contents)
-
-
-@pytest.mark.usefixtures('azure_service')
-def test_upload_without_timestamp(test_client: testing.FlaskClient,
-                                  sample_salt: str,
-                                  sample_zipfile_signature: str,
-                                  sample_zipfile: io.BytesIO) -> None:
-
-    post_return = test_client.post('/upload_file', data=dict(salt=sample_salt, signature=sample_zipfile_signature,
-                                                             file=(sample_zipfile, 'zipfile')))
-    assert post_return.status_code == HTTPStatus.BAD_REQUEST
```

Comment:
I think we still need these two tests - otherwise `pytest tests/test_endpoint.py` will not do a full test of `endpoint.py`. I know we essentially duplicate this testing in `tests/test_post_endpoint.py` but I don't think it's good to rely on that file to ensure that `endpoint.py` is tested. Thoughts?

Reply:
Good catch! I agree and have put them back in, which does mean one azure reference needs to sneak back in 😭 I think there's value in refactoring these eventually so that the endpoint tests don't rely on actually uploading to azure, since we test that functionality in azure_utils already, but for now this will have to stay!